src/status_im2/contexts/communities/actions/accounts_selection/view.cljs

```diff
@@ -41,6 +42,7 @@
                                          :accounts
                                          vals
                                          (map #(assoc % :customization-color (:color %))))]
+
```

Comment:
@FFFra I know you haven't written this code, but this subscription is way to broad, it's subscribing to the whole wallet and pulling just accounts out.
This means that anything that changes in `wallet` (regardless of whether `:accounts` is impacted), it will re-render the whole thing.

All of this should be in a subscription:

```
        accounts                    (->> (rf/sub [:wallet])
                                         :accounts
                                         vals
                                         (map #(assoc % :customization-color (:color %))))]
```
it should look something like:

`accounts-with-customization-color (rf/sub [:wallet/accounts-with-customization-color])` (or whatever naming, the idea is that all the work should be in a subscription and that should be at least a layer-2 subscription, data manipulation in the view should be kept to a minimum.

As a side node, this is a bit an issue with having nested structs in the database, they can help and they are neater, but it's more prone to error, so we should always be careful when we subscribe to make sure we only subscribe to what is strictly needed for the view and no more.

Reply:
@FFFra - can you explain why we need to add customization-color here?

Reply:
+ 1 on this.

Reply:
That's my code 😬.

Reply:
I have the same thing in my PR too.

Reply:
I can update my PR with a subscription like @cammellos suggested and once that's merged @FFFra can make use of that. Or should we go with a follow-up issue approach?

Reply:
up to you, since it's not going to be used and we can merge things quite quickly, either are fine, best to merge as quick as possible since it's all behind a flag, and follow up, to avoid conflicts, but up to you, as long as we don't forget.

Reply:
I think followup is the best approach to avoid conflicts. I have created this issue and assigned myself, will raise a PR after this one is merged.

@FFFra you can skip this suggestion for now.